ConwaysGameOfLife.Tests/UniverseShould.cs

```diff
@@ -66,11 +66,11 @@ public void WhenInitialized_FindCellStateOfCell(CellState state, int x, int y)
         [InlineData("XXX", "OXX", "XXX", 1, 1, 1 )]
         [InlineData("OOO", "OOO", "OOO", 1, 1, 8)]
         [InlineData("OXX", "XXX", "XXO", 1, 1, 2)]
-        public void WhenInitialized_ReturnsCountOfLiveNeighbours(string col1, string col2, string col3, int x, int y, int expected) 
+        public void WhenInitialized_ReturnsCountOfLiveNeighbours(string row1, string row2, string row3, int x, int y, int expected) 

```

Comment:
not testing the `Universe` constructor here
```suggestion
        public void ReturnCountOfLiveNeighbours_GivenCoordinates(string row1, string row2, string row3, int x, int y, int expected) 
```